spec/facter/facter_spec.rb

```diff
@@ -212,6 +212,15 @@ def mock_collection(method, os_name = nil, error = nil)
       expect(Facter.value('os.name')).to eq('Ubuntu')
     end
 
+    it 'call value twice and and resolved once' do
+      mock_fact_manager(:resolve_fact, [os_fact])
+
+      allow(fact_collection_spy).to receive(:value).with('os.name').and_return('Ubuntu')
+
+      expect(Facter.value('os.name')).to eq('Ubuntu')
+      expect(Facter.value('os.name')).to eq('Ubuntu')
```

Comment:
suggest adding the expectation on:

```
expect(fact_collection_spy).to receive(:value).with('os.name').and_return('Ubuntu')
```

and then calling `Facter.value('os.name')` twice, so it's clearer what's being tested. Also won't need to change rubocop.

Reply:
Yup that did the track! 👍 No rubocop exception required.